src/sdk/bucketspace_fs.js

```diff
@@ -278,6 +276,9 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
 
     async create_bucket(params, sdk) {
         return bucket_semaphore.surround_key(String(params.name), async () => {
+            if (!sdk.requesting_account.allow_bucket_creation) {
+                throw new RpcError('BUCKET_CREATION_NOT_ALLOWED');
```

Comment:
please change to 'throw new RpcError('UNAUTHORIZED', 'Not allowed to create new buckets');' 
found it in bucket_server :)